README.md

```diff
@@ -1,2 +1,79 @@
 # learn-ai
-AI Learn Test Repo
+
+![CI Workflow](https://github.com/mitodl/learn-ai/actions/workflows/ci.yml/badge.svg)
+
+This application provides backend API endpoints to access various AI chatbots.
+
+**SECTIONS**
+
+1. [Initial Setup](#initial-setup)
+2. [Configuration](#configuration)
+3. [Committing & Formatting](#committing--formatting)
+4. [Sample Requests](#sample-requests)
+
+## Initial Setup
+
+Learn-AI follows the same [initial setup steps outlined in the common OL web app guide](https://mitodl.github.io/handbook/how-to/common-web-app-guide.html).
+Run through those steps **including the addition of `/etc/hosts` aliases and the optional step for running the
+`createsuperuser` command**.
+
+This app runs locally on port 8002.
+
+You can start it by running `docker compose up`
+
+## Configuration
+
+Configuration can be put in the following files which are gitignored:
+
+```
+mit-learn/
+  ├── env/
+  │   └── backend.local.env (provided only to frontend containers)
+  └── .env (legacy file)
+```
+
+You will need at minimum the following environment variables to run locally:
+
+```
+OPENAI_API_KEY=<your_openai_api_key>
+MITOL_APP_BASE_URL=http://ai.open.odl.local:8002
```

Comment:
We could put this in `backend.env`, the committed defaults, then user only needs to specify the secret stuff.